zerver/management/commands/register_server.py

```diff
@@ -58,16 +58,10 @@ def handle(self, *args: Any, **options: Any) -> None:
             raise CommandError(
                 "Missing zulip_org_key; run scripts/setup/generate_secrets.py to generate."
             )
-        if settings.PUSH_NOTIFICATION_BOUNCER_URL is None:
-            if settings.DEVELOPMENT:
-                settings.PUSH_NOTIFICATION_BOUNCER_URL = (
-                    settings.EXTERNAL_URI_SCHEME + settings.EXTERNAL_HOST
-                )
-            else:
-                raise CommandError(
-                    "Please uncomment PUSH_NOTIFICATION_BOUNCER_URL "
-                    "in /etc/zulip/settings.py (remove the '#')"
-                )
+        if not settings.USE_PUSH_NOTIFICATIONS_SERVICE:
+            raise CommandError(
+                'Please add "mobile_push" to ZULIP_SERVICES ' "in /etc/zulip/settings.py"
```

Comment:
The quoting on this is a bit odd.

Reply:
You meant the combined strings with one wrapped in `'` and the other in `"`, in a single one, right? Fixed that. `"mobile_push"` is intentional, unless we prefer to just write `mobile_push`, even though the admin has to add it as a string when configuring?

Reply:
Yeah, that's what I meant.